measure/endurance-snapshot

```diff
@@ -475,8 +467,22 @@ fi
 echo "- Disk usage"
 LC_ALL=C df -k -P > $path/df
 
-echo "- statefs"
-dump_statefs_value namespaces/Screen/Blanked
+echo "- mce"
+# Stores the output from mcetool display stats
+# If mcetools isn't available, no file is generated
+mce_state=$(mcetool --get-display-stats=machine 2>/dev/null)
```

Comment:
I'd dump the tool output to file as-is, and do all the parsing later in the perl script...

(If "mcetool" reports some other useful information that could be added to endurance-plot graphs, in addition to the display stats, you could collect it here too.)

$path/mce_display_stats could be better file name.

Reply:
"mce_state" could indeed be confusing filename (mce tracks lots of various states, but there is no mce state -> using mce_display_stats would be more descriptive).

There is a lot of state info that could be obtained via "mcetool --status", but it includes also things like current setting values - it might be too much and too slow due to synchronous ipc. But for example battery/charger info might be interesting to add later on?

Reply:
Also, using shell script to parse into intermediate format and then using perl later on anyway -> I'd too dump it as-is unless there is some reason not to.

Reply:
But as long as it works, up to you @llewelld

Reply:
The latest push changes this to do all of the processing in the post-proc step.